ArduPlane/quadplane.cpp

```diff
@@ -658,12 +659,15 @@ bool QuadPlane::setup(void)
         SRV_Channels::set_default_function(CH_8, SRV_Channel::k_motor4);
         SRV_Channels::set_default_function(CH_11, SRV_Channel::k_motor7);
         AP_Param::set_frame_type_flags(AP_PARAM_FRAME_TRICOPTER);
+        coptar_tailsitter_supported = false;
```

Comment:
Does the deserve a different PR?  It's not related to moving strings around?

Also... `coptar`?!

Reply:
probably... I'll have to think about how to split those changes into a separate commit though
Don't like the term "copter tailsitter"? I'm not sure whether to blame me or @IamPete1 :)

Reply:
removed those changes and squashed